src/Elasticsearch.Net/Serialization/DiagnosticsSerializerProxy.cs

```diff
@@ -32,12 +32,16 @@ public SerializerRegistrationInformation(Type type, string purpose)
 		public override string ToString() => _stringRepresentation;
 	}
 
+	//TODO this no longer needs to be IInternalSerializerWithFormatter
```

Comment:
Does `DiagnosticsSerializerProxy` still need to implement `IInternalSerializerWithFormatter`?

Reply:
Remove it instead since its internal